modules/distr-tracing-tempo-config-query-frontend.adoc

```diff
@@ -8,60 +8,139 @@
+|type: integer; format: int32
+|
+
+|`tolerations`
+|Specifies component-specific pod tolerations.
+|type: array
 |
 |===
 
 
-.Configuration parameters passed to Query
+.Configuration parameters for the Query Frontend
 [options="header"]
-[cols="l, a, a, a"]
 |===
 |Parameter |Description |Values |Default value
 
-|spec:
-  query:
-    options: {}
-|Configuration options that define the Query service.
+|`component`
+|Configuration of the Query Frontend component.
+|type: object
+|
+
+|`nodeSelector` (inside component)
+|The simple form of the node selection constraint.
+|type: object
+|
+
+|`replicas` (inside component)
+|Specifies the number of replicas to be created for the component.
+|type: integer; format: int32
+|
+
+|`tolerations` (inside component)
+|Specifies component-specific pod tolerations.
+|type: array
+|
+
+|`jaegerQuery`
+|Specifies the options specific to Jaeger Query.
+|type: object
+|
+
+|`enabled` (inside jaegerQuery)
+|When `enabled`, creates the Jaeger Query component,`jaegerQuery`.
+|type: boolean
+|
+
+|`ingress` (inside jaegerQuery)
+|Specifies the options for the Jaeger Query Ingress.
+|type: object
+|
+
+|`annotations` (inside jaegerQuery.ingress)
+|Specifies the annotations of the Ingress object.
+|type: object
+|
+
+|`host` (inside jaegerQuery.ingress)
+|Specifies the hostname of the Ingress object.
+|type: string
+|
+
+|`ingressClassName` (inside jaegerQuery.ingress)
+|The name of an IngressClass cluster resource. Defines which Ingress controller serves this Ingress resource.
+|type: string
 |
+
+|`route` (inside jaegerQuery.ingress)
+|Specifies OpenShift Route specific options.
+|type: object
 |
 
-|options:
-  log-level:
-|Logging level for Query.
-|`debug`, `info`, `warn`, `error`, `fatal`, `panic`
+|`termination` (inside jaegerQuery.ingress.route)
+|The termination type. The default is `edge`.
+|type: string (enum: insecure, edge, passthrough, reencrypt)
 |
 
-|options:
-  query:
-    base-path:
-|You can set the base path for all tempo-query HTTP routes to a non-root value: for example, `/tempo` will cause all UI URLs to start with `/tempo`. This can be useful when running `tempo-query` behind a reverse proxy.
-|`/<path>`
+|`type` (inside jaegerQuery.ingress)
+|Specifies the type of Ingress for the Jaeger Query UI. The following types are supported: `ingress`, `route`, and `none`.
+|type: string (enum: ingress, route)
 |
+
+|`monitorTab` (inside jaegerQuery)
+|Specifies the monitor tab configuration.
+|type: object
+|
+
+|`enabled` (inside jaegerQuery.monitorTab)
+|Enables the monitor tab in the Jaeger console. The `PrometheusEndpoint` must be configured.
+|type: boolean
+|
+
+|`prometheusEndpoint` (inside jaegerQuery.monitorTab)
+|Configures the endpoint to the Prometheus instance that contains the span RED metrics. For example, `+https://thanos-querier.openshift-monitoring.svc.cluster.local:9091+`.
+|type: string
+|
+
 |===
 
 .Sample Query configuration
```

Comment:
I know this is outside the scope of the PR but this is also confusing here. Is this a Jaeger Query?

Reply:
This page is part of the docs for Tempo (replacement for Jaeger), so I assume it's sufficiently clear from the overall context.

Reply:
Good point. I'll check with the SME nevertheless.

Reply:
Thank you. I've checked with the SMEs and am rewriting this.